code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -108,5 +108,6 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/reagent_containers/food/snacks/canned/peaches/maint = 1,
 	/obj/item/reagent_containers/food/drinks/bottle/homemaderum = 1,
 	/obj/item/reagent_containers/food/snacks/grown/poppy = 10,
+	/obj/item/throwing_star = 1,
```

Comment:
why add throwing stars to the loot pool?

Reply:
they no longer give illegal tech, and it is a single item